The code uses iostream, not stdio.h. Failing to include iostream
causes "error: no member named 'cout' in namespace 'std'".
--- src/VoiceRecorder.cpp.orig	2009-08-29 15:39:06.000000000 -0500
+++ src/VoiceRecorder.cpp	2018-10-17 21:46:33.000000000 -0500
@@ -21,7 +21,7 @@
 
 #include "VoiceRecorder.h"
 #include <assert.h>
-#include <stdio.h>
+#include <iostream>
 #include "Order.h"
 #include "Utilities.h"
 
